.github/actions/setup-kubernetes/action.yaml

```diff
@@ -110,10 +110,22 @@ runs:
       kind: InitConfiguration
       nodeRegistration:
         criSocket: "unix:///var/run/crio/crio.sock"
+        kubeletExtraArgs:
+          runtime-cgroups: /system.slice/crio.service
       ---
-      kind: KubeletConfiguration
       apiVersion: kubelet.config.k8s.io/v1beta1
-      cgroupDriver: cgroupfs
+      kind: KubeletConfiguration
+      cgroupDriver: systemd
+      cgroupRoot: /
+      systemCgroups: /system.slice
+      kubeletCgroups: /system.slice/kubelet.service
+      enforceNodeAllocatable:
+      - pods
+      - system-reserved
```

Comment:
this was commented out before, did you add it on purpose?

Reply:
yep, trying it out on its own still a fail because GH CI is missing a cgroup path that is present on my brand new 20.04 ubuntu VM :shrug: (going back)

Reply:
(I half forgot whether this failed on its own yesterday with just the storage, already reverted to what was commented out before)